.github/workflows/natsjs.yml

```diff
@@ -0,0 +1,48 @@
+name: NATS.js CI
+
+on:
+  push:
+    tags:
+      - '*'
+    branches:
+      - '*'
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    name: ${{ matrix.config.kind }} ${{ matrix.config.os }}
```

Comment:
Doesn't look like the name is actually used in the list [here](https://github.com/nats-io/nats.js/actions/runs/136309302). In the NATS Server, we don't have name here, and what is displayed is the name of the job "test" followed by the Go version, for instance "test 1.13", without us having to add anything. Maybe that would work here too?